spec/features/recruiter/recruiter_can_navigate_to_indexes_from_player_profile_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.feature "recruiter can use navbar links" do
+    before(:each) do
+      @recruiter = User.create(type: "Recruiter",
+                          first_name: "test",
+                          last_name: "test",
+                          password: "123")
+      @recr_profile = RecruiterProfile.create(recruiter_id: @recruiter.id)
+      @player = User.create(type: "Player", first_name: "Henry", last_name: "Ford", password: "123")
+      @profile = PlayerProfile.create(player_id: @player.id, guardian_phone: "16073426730")
+      @facility = Facility.create(name: "test")
+      @game = Game.create!(facility_id: @facility.id, status: 0)
+      @team = Team.create(name: "Broncos")
+      @team.players << @player
+      PlayerStat.create(points: 40, fouls: 20, player_profile: @profile, game_id: @game.id)
+      Prospect.create(recruiter_profile_id: @recr_profile.id, player_profile_id: @profile.id)
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@recruiter)
+    end
+
+  scenario "from player profile show" do
+    visit player_profile_path(@profile)
+
+    expect(page).to have_content(@player.first_name)
+    click_on "Athletes"
+
+    expect(page).to have_content("All Athletes")
+    visit player_profile_path(@profile)
+
+    click_on "Recruiters"
```

Comment:
@mdevoe12 a player should never be able to see a recruiter list, nor the recruiter's profile, even after contacted by a recruiter.